app/uk/gov/hmrc/nisp/controllers/auth/ExcludedAuthActionImpl.scala

```diff
@@ -46,7 +46,7 @@ class ExcludedAuthActionImpl @Inject() (
     implicit val hc: HeaderCarrier    = HeaderCarrierConverter.fromRequestAndSession(request, request.session)
     implicit val ec: ExecutionContext = executionContext
 
-    authorised(ConfidenceLevel.L200)
+    authorised()
```

Comment:
I don't think this needs to be removed since the exclusion auth action doesn't use pertax within the exclusionController